lib/matplotlib/testing/__init__.py

```diff
@@ -9,6 +9,8 @@
 import subprocess
 import sys
 
+import pytest
```

Comment:
I don't think this is allowed at a top-level in this module.

Reply:
Done